shell/platform/android/io/flutter/view/FlutterNativeView.java

```diff
@@ -65,9 +72,9 @@ public FlutterPluginRegistry getPluginRegistry() {
         return mPluginRegistry;
     }
 
-    public void attachViewAndActivity(FlutterView flutterView, Activity activity) {
-        mFlutterView = flutterView;
-        mPluginRegistry.attach(flutterView, activity);
+    public void attachViewAndActivity(NativeViewHelper nativeViewHelper, Activity activity) {
```

Comment:
Here again, we can't change a public API.

Reply:
Again NativeViewHelper relaxes the API so if anyone is calling it with FlutterView (or anything that extends FlutterView), the source code does not have to change.